### ISSUE TITLE
When a Signal event is added to the workflow, any activities added later cannot be dragged

### ISSUE THREAD

**Issue by hyzx86** [OrchardCMS/OrchardCore#7248]
This problem will only be triggered in the Chinese environment, but the English environment is normal

![微信截图_20201011235043](https://user-images.githubusercontent.com/15613121/95683255-a0b89100-0c1c-11eb-85c3-4b222bd26b11.png)

**Comment by Skrypt** [comment]
You get any javascript error reported in your browser dev console?

**Comment by hishamco** [comment]
May be an encoding issue, but as Jasim said check whether errors has been reported in the browser console

**Comment by hyzx86** [comment]
> You get any javascript error reported in your browser dev console?

No , There is no exception in the console。
![wf](https://user-images.githubusercontent.com/15613121/95685694-1546fc00-0c2c-11eb-9a37-6c695a09657c.gif)

**Comment by Skrypt** [comment]
Press F12 and go to Console tab in Chrome dev tool.

**Comment by hyzx86** [comment]
> Press F12 and go to Console tab in Chrome dev tool.

As you can see on the right side of the screenshot above, my resource file was loaded locally, and there was no exception during the loading process. I tried it and it was normal in English.

**Comment by Skrypt** [comment]
Yeah sorry I did not see it first time because it was in chinese. All good then.
Is there a difference in the HTML elements rendered for these signal "cards" when you create one with an english title and one with a chinese title?

**Comment by hyzx86** [comment]
> Yeah sorry I did not see it first time because it was in chinese. All good then.
> Is there a difference in the HTML elements rendered for these signal "cards" when you create one with an english title and one with a chinese title?
There was no abnormality. I even tried to debug the JS file, but no abnormality was found. But because it is compressed, I don't know much about it, so I didn't study it in depth.

The following is a demo in different languages

![wfSignal2](https://user-images.githubusercontent.com/15613121/95685994-d0bc6000-0c2d-11eb-9874-df33c0e07b49.gif)

**Comment by Skrypt** [comment]
Ok, so there is no "Title" added to the Signal event. So, it renders a chinese title meaning "Signal" because we made these things translatable or you did?

**Comment by hyzx86** [comment]
![image](https://user-images.githubusercontent.com/15613121/95686096-6c4dd080-0c2e-11eb-88a7-c46165fb712f.png)
Just like this, even if the title is added, it won’t work

**Comment by Skrypt** [comment]
Ok, thanks, I will try to repro locally. I can't say it's because of an encoding issue as these are rendered correctly. Looks more like a javascript issue to me. But there is no console error reported. Really strange.

**Comment by hyzx86** [comment]
The simplest example: I first add a signal named "s1", and then add a signal named "s2", at this time s2 or other subsequently added nodes cannot be dragged
![Uploading image.png…]()

**Comment by Skrypt** [comment]
Maybe it is something that @sfmskywalker fixed already in Elsa-Core?

**Comment by hishamco** [comment]
I may try this in Arabic, this will be a first time I will use Workflows in action ;)

**Comment by hyzx86** [comment]
> Maybe it is something that @sfmskywalker fixed already in Elsa-Core?

Multi-language is a feature in Orchardcore, I think this problem should come from the code in OrchardCore

**Comment by hishamco** [comment]
> Multi-language is a feature in Orchardcore, I think this problem should come from the code in OrchardCore

Globalization & Localization is cool, but sometimes let you crazy ;) till now we didn't confirm if the issue caused from OC itself, we will try to reproduce the issue first

**Comment by Skrypt** [comment]
> Multi-language is a feature in Orchardcore, I think this problem should come from the code in OrchardCore

Yeah, I know I did code a lot of that culture module but did not code anything related with Workflow much. I'm pretty sure Elsa-Core has also some localization features. And probably uses also the same javascript base libraries to render it's workflow elements.

**Comment by sfmskywalker** [comment]
> Maybe it is something that @sfmskywalker fixed already in Elsa-Core?

I wasn't aware of this issue, so probably the Elsa designer has the same bug 😅

**Comment by justlucky** [comment]
Reason found, translation file error.

https://github.com/OrchardCMS/OrchardCore.Translations/blob/dev/Localization/zh-CN/OrchardCore.Workflows.po

![image](https://user-images.githubusercontent.com/247791/101971627-cb02eb00-3c6d-11eb-83c8-efc4b4846f04.png)

**Comment by Skrypt** [comment]
Someone fixed this? 
/cc @agriffard

**Comment by hyzx86** [comment]
Hi @agriffard  , I just updated the Chinese translation on Crowdin and fixed this problem. Could you update it to OrchardCore?

**Comment by agriffard** [comment]
I updated the .po files and built the translations.
You can download them from Crowdin and install them locally to test (but the nuget package won't be updated, it is only updated after a Release).

**Comment by hyzx86** [comment]
Will the po file in the source code repository be updated as well? I haven't seen its update yet.
`https://github.com/OrchardCMS/OrchardCore.Translations/blob/dev/Localization/zh-CN/OrchardCore.Workflows.po`

**Comment by hyzx86** [comment]
Although it seems that there are no white space characters,
But in some cases it still cannot be dragged
What's even stranger is that you can't switch prophecies now. . Even if I turn off the localization module
